subgraph.template.yaml

```
@@ -288,5 +286,4 @@ templates:
           handler: handleDefaultDestinationChainSet
         - event: CustomDestinationChainSet(indexed address,indexed uint256)
           handler: handleCustomDestinationChainSet
-      file: ./src/Task.ts
-
+      file: ./src/Task.ts
```

Don't forget the new line :)

```suggestion
      file: ./src/Task.ts

```